stream-chat-android-client/src/test/java/io/getstream/chat/android/client/parser/EventArguments.kt

```diff
@@ -283,7 +286,8 @@ internal object EventArguments {
         Arguments.of(createNotificationMutesUpdatedEventStringJson(), notificationMutesUpdatedEvent),
         Arguments.of(createNewMessageEventStringJson(), newMessageEvent),
         Arguments.of(createUnknownEventStringJson(), unknownEvent),
-        Arguments.of(createUnknownEventStringJson("some.unknown.type"), otherUnknownEvent)
+        Arguments.of(createUnknownEventStringJson("some.unknown.type"), otherUnknownEvent),
+        Arguments.of(createMarkAllReadEventStringJson(), markAllReadEvent)
```

Comment:
Let's add a trailing comma here, to avoid these kinds of diffs in the future when adding new items here

![image](https://user-images.githubusercontent.com/12054216/98646838-44888000-2334-11eb-9bfa-93ab8ee08422.png)

Reply:
Sure.